src/AcmSearchbar/AcmSearchbar.tsx

```diff
@@ -143,7 +143,7 @@ export function AcmSearchbar(props: AcmSearchbarProps) {
                 className={'help-button'}
                 onClick={toggleInfoModal}
                 noVerticalAlign
-                title={'Open help modal'}
+                title={'Open Help Modal'}
```

Comment:
@dislbenn thanks for reaching out over Slack! I recommend that the message reads: 

Open help modal